swlor_2da/phenotype.2da

```diff
@@ -1,12 +1,38 @@
 2DA V2.0
 
-    Label       Name        DefaultPhenoType
-0   Normal      2223        0
-1   Skinny      ****        0
-2   Large       2225        0
-3   Normal_M    111015      0
-4   ****        ****        0
-5   Large_M     111016      2
-6   N_Joust_M   111017      0
-7   ****        ****        0
-8   L_Joust_M   111018      2
+      Label     	Name   		DefaultPhenoType
+0     Normal    	2223   		0
+1     Skinny    	****   	     	0
+2     Large     	2225   	     	0
+3     Normal_M  	111015 		0
+4     ****      	****   	     	0
+5     Large_M   	111016 	     	2
+6     N_Joust_M 	111017 	     	0
+7     ****      	****   	     	0
+8     L_Joust_M 	111018 	     	2
+9     NULL7       	****		0
+10    NULL8       	****		0
+11    NULL9       	****		0
+12    NULL10      	****		0
+13    NULL11      	****		0
+14    NULL12      	****		0
+15    Kensai      	16797617	0
```

Comment:
Do we actually have the ACP animations in place? If not, can you remove the lines pertaining to them? It'll be confusing if they don't actually exist but we have 2DA entries for them